x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -191,7 +191,10 @@ export const resetAllRulesIdleModalTimeout = () => {
 
 export const changeRowsPerPageTo = (rowsCount: number) => {
   cy.get(PAGINATION_POPOVER_BTN).click({ force: true });
-  cy.get(rowsPerPageSelector(rowsCount)).click();
+  cy.get(rowsPerPageSelector(rowsCount))
```

Comment:
Click until we see the desired effect, the effect here being "element is not visible," indicating that the click was received and thus the popover was hidden.

In this way, we avoid the issue at hand, where we were trying to click an element as it was animated/redrawing. 👍